alibi/utils/distance.py

```diff
@@ -161,10 +162,10 @@ def abdm(X: np.ndarray,
 
 
 def multidim_scaling(d_pair: dict,
+                     feature_range: tuple,
```

Comment:
`feature_range` is never checked if it can be `None` and all uses of it have non-trivial `feature_range`, therefore it should be a required argument. This also required refactoring to bring it up a few positions in the function signature.